src/components/Footer.js

```diff
@@ -0,0 +1,23 @@
+import React, { useContext } from 'react';
+
+import { MapCenterContext } from '../context/MapCenterContext';
+
+import Nav from './Nav';
+import ShareMyLocationButton from './ShareMyLocationButton';
+import FlyToMeButton from './FlyToMeButton';
+
+const Footer = () => {
```

Comment:
Good idea 😄 ! the Footer section could be useful if we want to create more things at the bottom of our app (like the app version).